tests/unit/date-mixin.spec.ts

```diff
@@ -0,0 +1,102 @@
+    expect(vm.dateToSimpleDate(new Date('2021-07-01 00:00:00 PDT'))).toBe('2021-07-01')
+    expect(vm.dateToSimpleDate(new Date('2021-07-01 23:59:59 PDT'))).toBe('2021-07-01')
+    // verify that Eastern is correctly converted to Pacific
+    expect(vm.dateToSimpleDate(new Date('2021-01-01 02:00:00 EST'))).toBe('2020-12-31')
+    expect(vm.dateToSimpleDate(new Date('2021-01-01 03:00:00 EST'))).toBe('2021-01-01')
+    expect(vm.dateToSimpleDate(new Date('2021-07-01 02:00:00 EDT'))).toBe('2021-06-30')
+    expect(vm.dateToSimpleDate(new Date('2021-07-01 03:00:00 EDT'))).toBe('2021-07-01')
+  })
+
+  it('returns correct values for numToSimpleDate()', () => {
+    expect(vm.numToSimpleDate(null)).toBeNull()
+    expect(vm.numToSimpleDate(1234567)).toBeNull()
+    expect(vm.numToSimpleDate(20210120)).toBe('2021-01-20')
+  })
+
+  it('returns correct values for compareDates()', () => {
+    expect(vm.compareDates(null, '2020-01-01', '<')).toBe(true)
+    expect(vm.compareDates('2019-12-31', null, '<')).toBe(true)
+    expect(vm.compareDates('2019-12-31', '2020-01-01', null)).toBe(true)
+    expect(vm.compareDates('2019-12-31', '2020-01-01', '<')).toBe(true)
+    expect(vm.compareDates('2020-01-01', '2019-12-31', '<')).toBe(false)
+    expect(vm.compareDates('2019-12-31', '2020-01-01', '>')).toBe(false)
+    expect(vm.compareDates('2020-01-01', '2019-12-31', '>')).toBe(true)
+    expect(vm.compareDates('2019-12-31', '2020-01-01', '==')).toBe(false)
+  })
+
+  it('returns correct values for earliestDate()', () => {
+    expect(vm.earliestDate(null, null)).toBeNull()
+    expect(vm.earliestDate(null, '2020-01-01')).toBe('2020-01-01')
+    expect(vm.earliestDate('2019-12-31', null)).toBe('2019-12-31')
+    expect(vm.earliestDate('2019-12-31', '2020-01-01')).toBe('2019-12-31')
+    expect(vm.earliestDate('2020-01-01', '2019-12-31')).toBe('2019-12-31')
+  })
+
+  it('returns correct values for latestDate()', () => {
+    expect(vm.latestDate(null, null)).toBeNull()
+    expect(vm.latestDate(null, '2020-01-01')).toBe('2020-01-01')
+    expect(vm.latestDate('2019-12-31', null)).toBe('2019-12-31')
+    expect(vm.latestDate('2019-12-31', '2020-01-01')).toBe('2020-01-01')
+    expect(vm.latestDate('2020-01-01', '2019-12-31')).toBe('2020-01-01')
+  })
+
+  it('returns correct values for simpleDateToDisplayDate()', () => {
+    expect(vm.simpleDateToDisplayDate(null)).toBeNull()
+    expect(vm.simpleDateToDisplayDate('123456789')).toBeNull()
+    expect(vm.simpleDateToDisplayDate('2020-01-01')).toBe('Jan 1, 2020')
+  })
+
+  it('returns correct values for apiToSimpleDateTime()', () => {
+    expect(vm.apiToSimpleDateTime('2021-01-01T00:00:00+00:00')).toBe('2020-12-31 04:00:00 PM') // PST
+    expect(vm.apiToSimpleDateTime('2021-07-01T00:00:00+00:00')).toBe('2021-06-30 05:00:00 PM') // PDT
+  })
+
+  // FUTURE: fix so this works in GH CI action
+  xit('returns correct values for simpleDateToApi()', () => {
```

Comment:
I'm not sure why this test succeeds locally but fails in GHA, but:
- I didn't change the method
- this is a new unit test
- I ran out of time